.github/workflows/plugins-ci.yml

```diff
@@ -110,6 +110,11 @@ jobs:
       matrix:
         node-version: ${{ fromJson(inputs.node-versions) }}
         os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
Can we use `macos-latest-large`?
```suggestion
        os: [macos-latest-large, ubuntu-latest, windows-latest]
```

Reply:
eventually it will migrate, as I would not count on it staying fixed. I think it's time we do this

Reply:
https://github.com/fastify/fastify/actions/runs/8874112954/job/24361249312

It doesn't seem possible to use `macos-latest-large`.